conda/recipes/ucxx/meta.yaml

```diff
@@ -163,6 +177,12 @@ outputs:
         - {{ pin_subpackage('libucxx', exact=True) }}
         - ucx
         - rmm =23.08
+        {% if cuda_major == "11" %}
+        - cudatoolkit {{ cuda_spec }}
```

Comment:
```suggestion
        - cudatoolkit
```

Reply:
Done in commit ( https://github.com/rapidsai/ucxx/pull/61/commits/44cab6ffa3eb2609a7341036a0cb5da17781f1b3 )